ext/bcmath/libbcmath/src/recmul.c

```diff
@@ -38,23 +38,23 @@
 
 
 #if SIZEOF_SIZE_T >= 8
-#  define BC_MUL_UINT_DIGITS 8
-#  define BC_MUL_UINT_OVERFLOW (BC_INT_T) 100000000
+#  define BC_MUL_INT_DIGITS 8
+#  define BC_MUL_INT_OVERFLOW (BC_INT_T) 100000000
```

Comment:
TBH I still don't really understand what those constants are for. The naming could be improved IMHO.

Reply:
Honestly, I can't think of a good name...

bcmul's new calculation logic efficiently adds and sums every 4/8 digits (32/64-bit) of a decimal number to `BC_INT_T`.

`BC_MUL_INT_DIGITS` defines the number of digits, and `BC_MUL_INT_OVERFLOW` means the smallest value that does not fit into `BC_MUL_INT_DIGITS`.

In a 64-bit environment, it is usually 8 digits and 100000000.

It is used to convert between bc_num and BC_INT_T and to adjust the digits of the calculation result.

Reply:
Are the following names accurate descriptions of what the constants represent?
```suggestion
#  define BC_VECTORIZED_MULTIPLICATION_VECTOR_SIZE 8
/* The boundary number is computed from BC_VECTORIZED_MULTIPLICATION_VECTOR_SIZE times BASE */
#  define BC_VECTORIZED_MULTIPLICATION_BOUNDARY_NUMBER (BC_INT_T) 100000000
```

Reply:
This is great! It perfectly describes what I meant. Thx!